tests/pipeline/test_buyback_auth.py

```diff
@@ -48,34 +39,39 @@
 
 buyback_authorizations_cases = [
     pd.DataFrame({
-        SHARE_COUNT_FIELD_NAME: [1, 15],
-        CASH_FIELD_NAME: [10, 20],
+        VALUE_FIELD_NAME: [1, 15],
+        VALUE_TYPE_FIELD_NAME: ["$M", "Mshares"],
+        BUYBACK_TYPE_FIELD_NAME: ["New", "Additional"],
         TS_FIELD_NAME: pd.to_datetime(['2014-01-05', '2014-01-10']),
         BUYBACK_ANNOUNCEMENT_FIELD_NAME: pd.to_datetime(['2014-01-04',
                                                          '2014-01-09'])
     }),
     pd.DataFrame(
-        columns=[SHARE_COUNT_FIELD_NAME,
-                 CASH_FIELD_NAME,
+        columns=[VALUE_FIELD_NAME,
+                 VALUE_TYPE_FIELD_NAME,
```

Comment:
`VALUE_TYPE` is the unit - e.g., shares, USD, CAD, etc. I think I was trying to make it generic enough to use for other loaders, but it's probably not clear what it is; I can rename it to `VALUE_UNIT` or something like that.

`BUYBACK_TYPE` is just the type of buyback (that's the actual name of the field from EventVestor) - e.g., New, Additional, Reduction, etc.